docs/devices-api/more-info/cordova.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Cordova"
+description: "A plugin to integrate with Metriport on Cordova - including Apple Health integrations."
+---
+
+<Info>
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
```

Comment:
NABD, but this text gives the impression this can only be used to connect with AppleHealth, but one could use the Cordova connector for Android/other providers as well, right?